job_templates/install_errata_-_katello_ansible_default.erb

```diff
@@ -0,0 +1,18 @@
+<%#
+name: Install Errata - Katello Ansible Default
+snippet: false
+model: JobTemplate
+job_category: Katello
```

Comment:
since REX won't be able to deal with multiple providers for the same job category, I'm afraid we'll need to create separate category called `Katello via Ansible` for now, otherwise all these job template would only be usable through Katello features.